frontend/src/main/components/Users/UsersTable.js

```diff
@@ -28,6 +29,29 @@ export default function UsersTable({ users}) {
     const toggleAdminCallback = async (cell) => { toggleAdminMutation.mutate(cell); }
 
 
+    //toggleDriver
+    function cellToAxiosParamsToggleDriver(cell) {
+        return {
+            url: "/api/admin/users/toggleDriver",
+            method: "POST",
+            params: {
+                id: cell.row.values.id
+            }
+        }
+    }
+
+    // Stryker disable all : hard to test for query caching
+    const toggleDriverMutation = useBackendMutation(
+        cellToAxiosParamsToggleDriver,
+        {},
+        ["/api/admin/users"]
+    );
+    // Stryker enable all 
+
+    // Stryker disable next-line all : TODO try to make a good test for this
+    const toggleDriverCallback = async (cell) => { toggleDriverMutation.mutate(cell); }
```

Comment:
The description of this PR says that it makes changes to the backend—but these are front end changes.

Is this a mistake, or does the description need to be updated?

Here's what is says right now:
> In this PR we add more information into the @APIParam fields in the UserController so better documentation appears in Swagger. It is based from PR https://github.com/ucsb-cs156-s23/proj-gauchoride-s23-5pm-4/pull/23, which in turn is based on PR https://github.com/ucsb-cs156-s23/proj-gauchoride-s23-5pm-4/pull/22, so merge in order. Closes https://github.com/ucsb-cs156-s23/proj-gauchoride-s23-5pm-4/issues/24

Reply:
If the other changes have already been merged to main, then maybe rebasing on main will fix things?

Or we can do some git surgery with git cherry-pick and git reset to turn this branch into one that just has the backend changes.